bumble/smp.py

```diff
@@ -1772,6 +1773,19 @@ def send_command(self, connection: Connection, command: SMP_Command) -> None:
         connection.send_l2cap_pdu(cid, command.to_bytes())
 
     def on_smp_pdu(self, connection: Connection, pdu: bytes) -> None:
+        # Parse the L2CAP payload into an SMP Command object
+        command = SMP_Command.from_bytes(pdu)
+        logger.debug(
+            f'<<< Received SMP Command on connection [0x{connection.handle:04X}] '
+            f'{connection.peer_address}: {command}'
+        )
+
+        # Security request is more than just pairing, so let applications handle them
+        if command.code == SMP_SECURITY_REQUEST_COMMAND:
```

Comment:
I suggest putting this code in a separate method called `Manager.on_smp_security_request_command` to follow the same pattern for command handlers as for the `Session` objects.